SCClassLibrary/Common/Control/Server.sc

```diff
@@ -360,7 +360,7 @@ Server {
 		var n = options.maxLogins ? 1;
 
 		numControl = options.numControlBusChannels div: n;
-		numAudio = options.numAudioBusChannels div: n;
+		numAudio = options.numPrivateAudioBusChannels div: n;
```

Comment:
Yes, seriously.

Based on [1], it seems that the bug dates back more than 10 years -- possibly all the way back to JMc's original implementation. [1] is my change to introduce ContiguousBlockAllocator, but I copied the (numAudioBusChannels, firstPrivateBus) mistake from the previous usage of PowerOfTwoAllocator!

This means, nobody ever ran out of audio buses -- not in some 14-15 years.

[1] https://github.com/supercollider/supercollider/commit/95065fd8f0a3bdd99c77200d8d928e15f5d3384d